client/editor/map_editor.cpp

```diff
@@ -107,33 +118,78 @@ void map_editor::check_open()
  */
 void map_editor::close()
 {
-  // Notify the server we are exiting edit mode
+  // Notify the server we are exiting edit mode and show all floating widgets
   struct connection *my_conn = &client.conn;
   dsend_packet_edit_mode(my_conn, false);
-
-  setVisible(false);
   queen()->mapview_wdg->show_all_fcwidgets();
   queen()->unitinfo_wdg->show();
+  setVisible(false);
 }
 
 /**
- * \brief Populate a combo box with the current list of players
+ * \brief Populate a combo box with the current list of players by nation
+ * name
  */
 void map_editor::update_players()
 {
   if (!players_done) {
+
+    // Ensure we can take all possible nations
+    send_chat_printf("/set allowtake hHaAoObd");
```

Comment:
This will get sent quite often once we add editing players. Maybe move this next to enabling the editor.

Reply:
Good call.